app/src/main/java/com/example/vlad/financemanager/PresenterMoneyCalculator.java

```diff
@@ -14,89 +14,94 @@
     private ModelMoneyCalculator model;
     private int modifyingOperationId;
 
-    public PresenterMoneyCalculator(IMoneyCalculation view){
+    public PresenterMoneyCalculator(IMoneyCalculation view) {
         this.view = view;
         model = new ModelMoneyCalculator();
     }
 
     /**
      * Method - updates the result int the calculator if the result is correct
-     * @param btnId Clicked button
+     *
+     * @param btnId       Clicked button
      * @param numberValue Text value of this button (0,1,2..9, .)
      */
-    public void calculatorBtnOnClick(int btnId, String numberValue){
-        switch (btnId){
+    public void calculatorBtnOnClick(int btnId, String numberValue) {
+        switch (btnId) {
             case R.id.calculatorBackButton:
                 view.setCalcResultText(model.clearLast());
                 break;
             case R.id.aditionCalculatorButton:
-                if(!model.mathOperationBtnClick(CalculatorOperations.ADD)){
+                if (!model.mathOperationBtnClick(CalculatorOperations.ADD)) {
                     model.clearNumber();
                     view.calculationErrorSignal();
                 }
                 view.setCalcResultText(model.getResultText());
                 break;
             case R.id.substractionCalculatorButton:
-                if(!model.mathOperationBtnClick(CalculatorOperations.SUB)){
+                if (!model.mathOperationBtnClick(CalculatorOperations.SUB)) {
```

Comment:
`if (!model.mathOperationBtnClick(CalculatorOperations.SUB)) {
                    model.clearNumber();	                 
                    view.calculationErrorSignal();	
}`
In every `case` this block of code is the same - can be extracted to a separate method